pkg/providers/instancetype/types.go

```diff
@@ -151,6 +152,10 @@ func computeRequirements(info *ec2.InstanceTypeInfo, offerings cloudprovider.Off
 	if info.ProcessorInfo != nil {
 		requirements.Get(v1beta1.LabelInstanceCPUManufacturer).Insert(lowerKabobCase(aws.StringValue(info.ProcessorInfo.Manufacturer)))
 	}
+	// EBS Bandwidth
+	if info.EbsInfo != nil && aws.StringValue(info.EbsInfo.EbsOptimizedSupport) != "unsupported" {
```

Comment:
nit: Is there a constant in the SDK that shows this is something that they consistently use or some documentation on the API that shows that they consistently use the word "unsupported" when the EBSOptimizedSupport isn't there?

Reply:
Looks like there is: https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#pkg-constants 

@NetanelK , can you add this in ?